battle-engine.js

```diff
@@ -495,7 +495,7 @@ BattlePokemon = (function () {
 		}
 		return null;
 	};
-	BattlePokemon.prototype.getMoveTargets = function (move, target) {
+	BattlePokemon.prototype.getMoveTargets = function (move, target, noRepeat) {
```

Comment:
Maybe resolve Pressure targets separately? Or just hardcode the Pressure stuff somewhere?